### ISSUE TITLE
Swiper is not visible on Android

### ISSUE THREAD

**Issue by novakben** [alexbrillant/react-native-deck-swiper#29]
Is there a way to apply justifyContent: 'center' to the card?
I can't center it vertically.

**Comment by pallavbakshi** [comment]
Hey @novakben, did you find a way man?

**Comment by novakben** [comment]
@pallavbakshi 
Yep, I hacked it by having the parent component contain it and positioned the parent component.

The only issue is that you have to plan ahead and make sure no other components are above that parent (like navigator usually is)- otherwise while swiping, the card can be under other components and you get a bad experience for the user.

Good luck!
